files/pt-br/web/api/document_object_model/how_to_create_a_dom_tree/index.html

```diff
@@ -4,7 +4,7 @@
 translation_of: Web/API/Document_object_model/How_to_create_a_DOM_tree
 original_slug: DOM/Referencia_do_DOM/How_to_create_a_DOM_tree
 ---
-<p>{{draft}}</p>
+<p></p>
```

Comment:
```suggestion
```

Reply:
I will remove that empty tag, thank you!